plenum/test/consensus/checkpoint_service/test_checkpoint_service_on_view_change.py

```diff
@@ -22,6 +22,7 @@ def test_do_nothing_on_view_change_started(internal_bus, checkpoint_service):
     assert old_data == new_data
 
 
+@pytest.mark.skip(reason='Stable checkpoint is not kept yet')
```

Comment:
I didn't get why we skipped this test. This is a new View Change logic not affecting legacy behavior, and it worked before. Why it doesn't work anymore?